x-pack/plugins/observability_onboarding/e2e/ftr_kibana.yml

```diff
@@ -0,0 +1,3 @@
+xpack.observability_onboarding.ui.enabled: true
+
+xpack.cloud.id: 'foo'
```

Comment:
We should probably set the cloud id value to something that matches the [expected format](https://github.com/elastic/kibana/tree/main/x-pack/plugins/cloud#cloudid)

Example: `eastus2.azure.elastic-cloud.com:9243$59ef636c6917463db140321484d63cfa$a8b109c08adc43279ef48f29af1a3911`

We may need to decode this id to get the ES deployment when we generate the `elastic-agent.yml`